posthog/cdp/templates/rudderstack/template_rudderstack.py

```diff
@@ -0,0 +1,118 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="alpha",
+    id="template-rudderstack",
+    name="Send data to RudderStack",
+    description="Send data to RudderStack",
+    icon_url="/api/projects/@current/hog_functions/icon/?id=rudderstack.com",
```

Comment:
This should be a proper icon now and can be included in the same folder as all the others.